examples/autocomplete.html

```diff
@@ -217,6 +217,10 @@
               ]);
             },
 
+          },
+          tagsModule: {
```

Comment:
To be similar to the other settings like `authors`. this would have to be a function that gets passed a `done()` callback, not an array -- the function can return an array with `done(array)` -- this allows it to easily become a remote call, and removes the asynchronous code from our codebase. But it looks like you've used a built-in call from Bloodhound here and are letting Bloodhound handle it? If so, that sounds all right; what do you think? Should we go for consistency or just use the Bloodhound system and keep it simple?